### PR TITLE
Fix prefixParameter frozen inheritance

### PR DESCRIPTION
**Pull request by dlakaplan**
changed prefixparameter creation, added test

Fixes https://github.com/nanograv/PINT/issues/1218

### CODE REVIEW

**Comment by dlakaplan** [comment]
@scottransom : I tested this with `pintk` and recentering `PEPOCH` in the pareditor and it no longer fits for F2 etc.

**Comment by dlakaplan** [comment]
What this doesn't do is deal with any more subtle changes like those @paulray was hinting at.

File: src/pint/models/parameter.py [hunk 1]
```diff
@@ -1253,9 +1253,9 @@ class prefixParameter:
     F22. Appropriate units will be inferred,
 
     To create a prefix parameter with the same prefix but different index, just
-    use the ``.new_param`` method. It will return a new prefix parameter with the
+    use the :meth:`pint.parameter.prefixParameter.new_param` method. It will return a new prefix parameter with the
     same setup but the index. Some parameters' unit and description will
-    be changed once the index has been changed. In order to get the right units
+    be changed once the index has been changed. The new parameter will not inherit the ``frozen`` status of its parent by default.  In order to get the right units
     and description, ``.unitTplt`` and ``.descriptionTplt`` should be provided. If
```
Comment (scottransom) [inline src/pint/models/parameter.py:1259]: Just noticed this, but I think the docs are outdated here.  Should be `unit_template` and `description_template`

**Comment by dlakaplan** [comment]
OK, that's fair.  I can see about updating the docs tomorrow.

**Comment by paulray** [comment]
I had a problem today with pintk where some parameters I had added like `F3 0` came into pint as unfrozen, which is definitely a bug.  I'm not sure if this PR fixes that. I'll try again when this is merged.

**Comment by scottransom** [comment]
Thanks, David K!

**Pull request merged**